benchmarks/mysql-kafka-cdc/benchi.yml

```diff
@@ -0,0 +1,104 @@
+infrastructure:
+  kafka:
+    compose: ../../shared/kafka/compose.yml
+  mongo:
+    compose: ../../shared/mysql/compose.yml
+
+tools:
+  kafka-connect:
+    compose:
+      - ../../shared/kafka-connect/compose.yml
+      - ./kafka-connect/compose.override.yml
+  conduit:
+    compose:
+      - ../../shared/conduit/compose.yml
+      - ./conduit/compose.override.yml
+
+metrics:
+  conduit_metrics:
+    collector: conduit
+    tools:
+      - conduit
+    settings:
+      url: http://localhost:8080/metrics
+  kafka_metrics_for_conduit:
+    collector: kafka
+    tools:
+      - conduit
+    settings:
+      url: http://localhost:7071/metrics
+      topics: [ "conduit.users" ]
+  kafka_metrics_for_kafka_connect:
+    collector: kafka
+    tools:
+      - kafka-connect
+    settings:
+      url: http://localhost:7071/metrics
+      topics: [ "conduit.conduit.users" ]
+  docker_metrics_for_conduit:
+    collector: docker
+    tools:
+      - conduit
+    settings:
+      containers:
+        - "benchi-conduit"
+  docker_metrics_for_kafka_connect:
+    collector: docker
+    tools:
+      - kafka-connect
+    settings:
+      containers:
+        - "benchi-kafka-connect"
```

Comment:
Might be a bit simpler:
```
  docker_metrics:
    collector: docker
    settings:
      containers:
        - benchi-conduit
        - benchi-kafka-connect
```
We can even call it a matter of preference.

Reply:
Doing it this way will also spawn only a single goroutine for the docker collector 👍